.github/workflows/web.yml

```diff
@@ -0,0 +1,197 @@
+# Copyright 2025 The Khronos Group Inc.
+# SPDX-License-Identifier: Apache-2.0
+name: KTX-Software Web CI
+
+# Seems no way to avoid duplicating this on logic in each .yml file.
+# See https://github.com/actions/starter-workflows/issues/245.
+on:
+  # Trigger the workflow on a pull request,
+  pull_request:
+
+  push:
+    # And on pushes to main, which will occur when a PR is merged.
+    branches:
+      - main
+    # Also trigger on push of release tags to any branch. Useful
+    # for testing release builds before merging to main.
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-*'
+    paths-ignore:
+      - .appveyor.yml
+      - .github/workflows/android.yml
+      - .github/workflows/check-mkvk.yml
+      - .github/workflows/docs.yml
+      - .github/workflows/formatting.yml
+      - .github/workflows/publish-pyktx.yml
+      - .github/workflows/windows.yml
+      - .travis.yml
+      - README.md
+      - CODE_OF_CONDUCT.md
+      - CONTRIBUTING.md
+      - LICENSE.md
+      - LICENSES
+      - RELEASE_NOTES.md
+      - REUSE.toml
+      - install-gitconfig*
+  # Allow manual trigger
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  #formatting:
+  #  uses: ./.github/workflows/formatting.yml
+  web:
+    # Shortcircuit and don't burn CI time when formatting will reject
+    #needs: formatting
+    strategy:
+      matrix:
+        include:
+        - name: Debug
+          os: ubuntu-22.04
+          generator: Ninja
+          arch: x86_64
+          compiler: clang
+          vk_sdk_ver: '1.3.290'
+          options: {
+            config: Debug,
+            doc: OFF, jni: OFF, loadtests: OpenGL, py: OFF, tools: OFF, tools_cts: OFF,
+            sse: OFF, opencl: OFF, wasm: ON
+          }
+        - name: Package
+          os: ubuntu-22.04
+          generator: Ninja
+          arch: x86_64
+          compiler: clang
+          vk_sdk_ver: '1.3.290'
+          options: {
+            config: Release,
+            doc: OFF, jni: OFF, loadtests: OpenGL, py: OFF, tools: OFF, tools_cts: OFF,
+            package: YES, sse: OFF, opencl: OFF, wasm: YES
```

Comment:
s/wasm: YES/wasm:ON/ to match the Debug case.

Reply:
Changed.